module/VuFind/src/VuFind/Search/Solr/Params.php

```diff
@@ -651,4 +651,20 @@ function ($part) {
 
         return $filter;
     }
+
+    /**
```

Comment:
You said this is no longer needed, but it's still here. Is it okay to remove these changes now?